articles/machine-learning/how-to-machine-learning-interpretability-automl.md

```diff
@@ -169,6 +193,48 @@ with open("myenv.yml","r") as f:
 
 ```
 
+### Write the Scoring Script
+
+Write the script that will be used to predict on your model and creating inference time explanations.
```

Comment:
Nit: tense agreement ("predict" vs "creating")
I propose something like that: "Write a script that loads your model and produces explanations based on a new batch of data."

Reply:
Changed. Done.